app/Helpers/Helper.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+use Illuminate\Support\Carbon;
```

Comment:
Please use eather Carbon\Carbon or Facades\Date.

Reply:
I'm not entirely sure about that. They explicitly state this in their documentation and the returntype of a `datetime`-cast is also `Illuminate\Support\Carbon`. https://laravel.com/docs/10.x/helpers#dates